cmd/chaininfo/chain_defaults.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package chaininfo
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var DefaultChainConfigs map[string]*params.ChainConfig
+
+func init() {
+	var chainsInfo []ChainInfo
+	err := json.Unmarshal(DefaultChainsInfoBytes, &chainsInfo)
+	if err != nil {
+		panic(fmt.Errorf("error initializing default chainsInfo: %w", err))
+	}
+	if len(chainsInfo) == 0 {
+		panic("Default chainsInfo is empty")
+	}
+	DefaultChainConfigs = make(map[string]*params.ChainConfig)
+	for _, chainInfo := range chainsInfo {
+		DefaultChainConfigs[chainInfo.ChainName] = chainInfo.ChainConfig
+	}
+}
+
+func CopyArbitrumChainParams(arbChainParams params.ArbitrumChainParams) params.ArbitrumChainParams {
+	return params.ArbitrumChainParams{
+		EnableArbOS:               arbChainParams.EnableArbOS,
+		AllowDebugPrecompiles:     arbChainParams.AllowDebugPrecompiles,
+		DataAvailabilityCommittee: arbChainParams.DataAvailabilityCommittee,
+		InitialArbOSVersion:       arbChainParams.InitialArbOSVersion,
+		InitialChainOwner:         arbChainParams.InitialChainOwner,
+		GenesisBlockNum:           arbChainParams.GenesisBlockNum,
+		MaxCodeSize:               arbChainParams.MaxCodeSize,
+		MaxInitCodeSize:           arbChainParams.MaxInitCodeSize,
+	}
+}
+
+func CopyChainConfig(chainConfig *params.ChainConfig) *params.ChainConfig {
```

Comment:
Yep, I did consider deserializing the json everytime the ...ChainConfig() functions are called but this adds unnecessary latency, sometimes in crucial parts of the codebase. 

Since these default configs should only be dependent on the arbitrum_chain_info.json I'll add a system test to check if default chain configs are equal to the ones returned by copying them, so that if in future we decide to use more fields of chainconfig and update the json file.... it forces us to also update the copy functions